src/lib.rs

```diff
@@ -130,6 +130,7 @@
 pub use crate::{
     framework::{parse_framework_args, run_bot, Bot, FrameworkArgs},
     game::*,
+    hive::{parse_hive_framework_args, run_hive, HiveFrameworkArgs, Hivemind},
```

Comment:
I think only exporting the hive module would be a better idea. I'm open to leaving these exposed in the root if anyone feels strongly about it. @Quetzal2 ?

Reply:
I liked it like that, having the same use as `framework`, thus the same way to use it. But if we say that in general, people will be more likely to create bots than hiveminds, and that hivemind is more a feature, I would see how it'd make sense to not expose it like that.
I think it's more a design choice, and both are ok to me. Anyways, if we go with the compact version, the user would just have to `use rlbot::hive::{…}` to have them exposed like again.